vision/image_simulator/create_patches.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+
+"""create_patches.py
+
+Create labeled samples for training the shape classification network.
+
+Author: Amit Aides, Ahmad Kiswani
+License: See attached license file
+"""
+
+from __future__ import division
+import AUVSItargets
```

Comment:
When I ran this, I had to go through a bunch of steps to get libraries installed for AUVSItargets, so please record whatever steps you took to get this library installed in the SETUP.md file (and feel free to fork the repos involved & make any necessary modifications, since it seems that aggdraw uses deprecated function calls)

Also set up a python include path that is relative to wherever the AUVSItargets gitmodule was installed.